web/modules/custom/d_partners/d_partners.install

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use \Drupal\node\Entity\Node;
+use \Drupal\file\Entity\File;
+
+/**
+ * Implements hook_install().
+ */
+function d_partners_install() {
+  // Create sample Partners content.
+  $partners = [
+    'Droptica' => [
+      'public://logo-droptica.svg' => 'https://droptica.pl',
+    ],
+    'Abventor' => [
+      'public://abventor-logotyp.svg' => 'https://www.abventor.com',
+    ],
+    'Famatech' => [
+      'public://famatech.svg' => 'https://www.famatech.pl',
+    ],
+    'Smartbees' => [
+      'public://smartbees_icon.svg' => 'https://smartbees.pl',
+    ],
+  ];
+
+  foreach ($partners as $title => $partner) {
+    foreach ($partner as $logoUri => $site) {
+      $files = \Drupal::entityTypeManager()
+        ->getStorage('file')
+        ->loadByProperties(['uri' => $logoUri]);
+      $file = reset($files);
+
+      if (!$file) {
+        $file = File::create([
+          'uri' => $logoUri,
+        ]);
+        $file->save();
+      }
+
+      $partner = Node::create(['type' => 'partners']);
+      $partner->set('title', $title);
+      $partner->set('body', 'Firma zajmująca się budową serwisów internetowych opartych o system Drupal');
```

Comment:
Dla każdego partnera dodałbym opis, ten przyjął jako domyślne, ale mogą być w przyszłości inne

Reply:
jak będą to się je zmieni w nodzie, moduł będzie cały czas już włączony więc *.install już sie nie odpali